portfolio/src/main/webapp/index.html

```diff
@@ -173,6 +173,21 @@ <h4 class="text-center">Post your thoughts!</h4>
             <button class="btn btn-primary" type="submit">Post</button>
           </form>
         </div>
+        <div class="card p-4 mb-4">
+          <form class="form-inline" id="comment-control">
+            <div class="form-group">
+              <button class="btn btn-primary" name="refresh">Refresh</button>
+            </div>
+            <div class="form-group ml-4">
+              <label for="numPerPage">Per Page:</label>
```

Comment:
Using `commentsPerPage` is more specific. You should change this everywhere it's used.
```suggestion
              <label for="commentsPerPage">Per Page:</label>
```